config/environment.js

```diff
@@ -17,7 +17,7 @@ module.exports = function(environment) {
       }
     },
 
-  firebase: {
+  firebase:  {
     apiKey: 'AIzaSyBqSKx49L3USjMlIfIhFKFNN3eiEh_roo0',
     authDomain: 'pstcc2017fieldwork.firebaseapp.com',
     databaseURL: 'https://pstcc2017fieldwork.firebaseio.com',
```

Comment:
[It's there](https://github.com/fdlane/fieldwork-worker/blob/02d1306c418e99eda2d2f78e6b78bfe2d9ed27dd/config/environment.js#L24). GitHub doesn't show it in this view because it is only showing the changes.